src/app/dynamic-components.ts

```diff
@@ -0,0 +1,488 @@
+import { Type } from "@angular/core";
+
+// TODO make a build script that looks for annotations and creates this
+export const dynamicComponents = new Map<string, () => Promise<Type<any>>>([
+  [
+    "Dashboard",
+    () =>
+      import("./core/dashboard/dashboard/dashboard.component").then(
```

Comment:
I think these functions are quite bad for the code coverage as none of them are executed in the tests.